chain/rosetta-rpc/src/adapters/validated_operations/delegate_action.rs

```diff
@@ -78,7 +77,6 @@ impl From<near_primitives::delegate_action::DelegateAction> for DelegateActionOp
             receiver_id: delegate_action.receiver_id.into(),
             max_block_height: delegate_action.max_block_height,
             public_key: (&delegate_action.public_key).into(),
-            operations: non_delegate_operations,
```

Comment:
Removing nested operations (otherwise the operations list appears twice in the output: once nested and once flat)